deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -208,6 +225,58 @@ Resources:
             - !Sub web/${FQDN}/
           IncludeCookies: true
 
+  LambdaEdgeBasicExecutionRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Effect: "Allow"
+            Principal:
+              Service:
+                - "lambda.amazonaws.com"
+                - "edgelambda.amazonaws.com"
+            Action:
+              - "sts:AssumeRole"
+      Path: "/"
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+  SPARedirectionLambda:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Code:
+        # This Lambda will take incoming web requests and adjust the request URI as appropriate.
```

Comment:
I would update this comment to describe what this specific Lambda is doing, since it is making different logic decisions than the one for archives-retention.

Reply:
done